content/techniques/caching.md

```diff
@@ -50,12 +50,20 @@ To add an item to the cache, use the `set` method:
 await this.cacheManager.set('key', 'value');
 ```
 
-You can also specify a TTL (expiration time) for this specific key, as follows:
+The default expiration time of the cache is 5 seconds.
+
+You can manually specify a TTL (expiration time) for this specific key, as follows:
 
 ```typescript
 await this.cacheManager.set('key', 'value', { ttl: 1000 });
 ```
 
+You can set the expiration time to be effectively infinite as follows:
```

Comment:
```suggestion
You can disable the cache by setting the ttl to `null`:
```

Reply:
These are two contrary sentences^ (infinite vs disabled)

Reply:
damn, you are right.. i meant it the other way, haha..

Reply:
corrected my suggestion